routers/api/v1/admin/adopt.go

```diff
@@ -43,9 +43,13 @@ func ListUnadoptedRepositories(ctx *context.APIContext) {
 	//     "$ref": "#/responses/forbidden"
 
 	listOptions := utils.GetListOptions(ctx)
+	if listOptions.Page <= 0 {
```

Comment:
Maybe it should be zero, then we can ignore page to use `-1`.

Reply:
Sorry, I don't understand your point. Why should the page be -1? if it's <= 0, it should be 1 usually, or do you want to change that it's 1 if it is <= 0?

Reply:
I mean how to ignore the pagination?

Reply:
Why should it ignore pagination, it's a list endpoint and should provide pagination like every other list endpoint?

Reply:
https://github.com/go-gitea/gitea/pull/18848/commits/95793162e00d40985dee516d59ba43adb8ed4706